src/status_im2/contexts/wallet/common/screen_base/create_or_edit_account/view.cljs

```diff
@@ -36,7 +36,7 @@
          {:customization-color account-color
           :size                80
           :emoji               account-emoji
-          :type                :default}]
+          :type                (if watch-only? :watch-only :default)}]
```

Comment:
this is the fix for: https://github.com/status-im/status-mobile/issues/17807

Reply:
🚀